### PR TITLE
[[ Merge 6.7.8 ]] 30-09-15

### PR DESCRIPTION
**Pull request by livecodesebastien**
Refactored changes:
- addition of `MCPlayer::setdontuseqt` in player-platform.cpp implementation
- addition of PlaybleDuration property for Android Native player
- Update to StringRef the Environment variable building update for iPhone (mbliphonedc.mm)

Conflicts:
    engine/src/mbliphonedc.mm
    engine/src/player-platform.cpp
    engine/src/stack2.cpp
    ide
    version

`ide` submodule and version will be updated in a subsequent commit

### CODE REVIEW

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 240bdd2

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 240bdd20ae82926bd5516882afa0ca5f47329702

**Comment by livecode-vulcan** [comment]
😞 test failure 240bdd20ae82926bd5516882afa0ca5f47329702
- try-community-linux-x86: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/545)
- try-community-mac: [failure](https://vulcan.livecode.com/builders/try-community-mac/builds/549)
- try-community-linux-x86_64: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/544)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/544)
- try-community-ios: [failure](https://vulcan.livecode.com/builders/try-community-ios/builds/553)
- try-community-win-x86: [pending](https://vulcan.livecode.com/builders/try-community-win-x86/builds/547)
- try-community-android-armv6: [failure](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/555)

File: engine/src/player-platform.cpp [hunk 1]
```diff
@@ -1951,6 +1953,21 @@ void MCPlayer::setlooping(Boolean loop)
...
+	// Weak link QT when setting a player's dontuseqt to false, if it is not already linked
+	if (!noqt && MCdontuseQT)
+	{
+		initialise_weak_link_QuickTime();
+		initialise_weak_link_QTKit() ;
+		initialise_weak_link_QuickDraw() ;
+	}
+}
+
 real8 MCPlayer::getplayrate()
```
Comment (livecodepanos) [inline engine/src/player-platform.cpp:1971]: I think the signature of setdontuseqt() has to be added in player-platform.h

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 9616d95

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 9616d95366a0a4ef0ab514e56a3b8ed58c7395f9

**Comment by livecode-vulcan** [comment]
😞 test failure 9616d95366a0a4ef0ab514e56a3b8ed58c7395f9
- try-community-linux-x86: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/547)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/551)
- try-community-linux-x86_64: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/546)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/546)
- try-community-ios: [pending](https://vulcan.livecode.com/builders/try-community-ios/builds/555)
- try-community-win-x86: [pending](https://vulcan.livecode.com/builders/try-community-win-x86/builds/549)
- try-community-android-armv6: [failure](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/557)

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 0b49cf8

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 0b49cf86a1bed5de13009902a4b5bd0fb3ddf3a0

**Comment by livecode-vulcan** [comment]
😎 test success 0b49cf86a1bed5de13009902a4b5bd0fb3ddf3a0
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/548)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/553)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/549)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/547)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/556)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/550)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/558)

**Pull request merged**